content/blog/2016/2016-01-07-official-code-of-conduct.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Jenkins Code of Conduct"
+author: rtyler
+---
+
+
+Over the past couple months we have been working on a long overdue
+link:/conduct[Code of Conduct] for the Jenkins project (meeting minutes
+link:http://meetings.jenkins-ci.org/jenkins-meeting/2015/jenkins-meeting.2015-12-09-19.01.html[here]
+and
+link:http://meetings.jenkins-ci.org/jenkins-meeting/2016/jenkins-meeting.2016-01-06-19.01.html[here]).
+Following in the footsteps of other projects like the
+link:https://www.apache.org/foundation/policies/conduct.html[Apache Software
+Foundation], link:https://golang.org/conduct[Go lang] and
+link:http://contributor-covenant.org/[countless others], we have adopted this
+code of conduct to help set guidelines for what behaviors are acceptable or
```

Comment:
I don't like the "what behaviors are acceptable or unacceptable" - even though redundant, I'd prefer "what behaviors are acceptable, and what behaviors are not" or something along those lines.